internal/k8s/controller.go

```diff
@@ -153,9 +154,105 @@ func isRouteUpdated(oldRoute, newRoute *routev1.Route) bool {
 	return false
 }
 
+func isNamespaceUpdated(oldNS, newNS *corev1.Namespace) bool {
+	if oldNS.ResourceVersion == newNS.ResourceVersion {
+		return false
+	}
+	oldLabelHash := utils.Hash(utils.Stringify(oldNS.Labels))
+	newLabelHash := utils.Hash(utils.Stringify(newNS.Labels))
+	if oldLabelHash != newLabelHash {
+		return true
+	}
+	return false
+}
+func AddIngressFromNSToIngestionQueue(numWorkers uint32, c *AviController, namespace string, msg string) {
+	ingObjs, err := utils.GetInformers().IngressInformer.Lister().ByNamespace(namespace).List(labels.Set(nil).AsSelector())
+	if err != nil {
+		utils.AviLog.Errorf("NS to ingress queue add: Error occured while retrieving ingresss for namespace:[%s]", namespace)
```

Comment:
If we return from here, then we can remove the `else` below.

Reply:
Done